### PR TITLE
[ja] fix incorrect pod-template-hash labels

### PR DESCRIPTION
**Pull request by Arhell**
ren PR https://github.com/kubernetes/website/pull/40752/files
/assign @atoato88
/assign @nasa9084

### CODE REVIEW

File: content/ja/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ Deploymentによって作成されたReplicaSetを管理しないでください
   コマンドの実行結果は以下のとおりです:
   ```shell
```
Comment (dipesh-rawat) [inline content/ja/docs/concepts/workloads/controllers/deployment.md:121]: The pod-template output changes are aligned with the 'en' PR

nit: Remove the unnecessary code type `shell` from all the output blocks in this file  _(Equivalent 'en' PR   #23270)_

Comment (Arhell) [reply content/ja/docs/concepts/workloads/controllers/deployment.md:121]: With this commit, I synchronize what is in the English version, the rest in another PR

Comment (Arhell) [reply content/ja/docs/concepts/workloads/controllers/deployment.md:121]: With this commit, I synchronize what is in the English version, the rest in another PR

**Pull request merged**